portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +77,35 @@ function deleteComment(comment) {
   params.append('id', comment.id);
   fetch('/delete-comment', {method: 'POST', body: params});
 }
+
+function loadMap() {
+  var sanFrancisco = {lat: 37.7749, lng: -122.4194};
+  const cafeMap = new google.maps.Map(
+    document.getElementById("map"),
+    {center: sanFrancisco, zoom: 12});
+
+  const philzMarker = new google.maps.Marker({
+    position: {lat: 37.793949, lng: -122.398062},
+    map: cafeMap, 
+    title: "Philz Coffee"
+  });
+
+  const sFrankMarker = new google.maps.Marker({
```

Comment:
maybe rename `sFrankMarker` to `saintFrankMarker` to be clearer?

ditto for `sMatchaMarker` --> `stonemillMatchaMarker` and `fBarrelMarker` --> `fourBarrelCoffeeMarker`